backend/src/test/java/heartbeat/repository/FileRepositoryTest.java

```diff
@@ -60,6 +60,10 @@ class FileRepositoryTest {
 
 	private static final String TEST_UUID = "test-uuid";
 
+	private static final int EXPIRED_DAYS = 7;
```

Comment:
add it into test/application.yaml

Reply:
I haven't implemented this part. I've tried many methods but none of them worked. You may need to solve it.